roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -750,6 +750,7 @@ class RooVectorDataStore : public RooAbsDataStore {
   mutable Double_t  _curWgtErr ;   // Weight of current event
 
   RooVectorDataStore* _cache ; //! Optimization cache
+  std::vector<RooVectorDataStore::RealVector*> _cacheUpdateList ; //! Pointers to the optimization cache
```

Comment:
We want to preallocate with `reserve()` at least 1024 elements, this would reduce the heap reallocations.